templates/index.html

```diff
@@ -197,9 +197,9 @@ <h1 class="title is-1"><a href="#showcase">#</a> Showcase</h1>
                 "has_webp": false,
             },
             {
-                "name": "Universal Android Debloater",
+                "name": "Universal Android Debloater Next Generation",
```

Comment:
The name will be too long, let's shorten it.

```suggestion
                "name": "Universal Android Debloater NG",
```